cmd/cli/mesh_list.go

```diff
@@ -62,17 +63,47 @@ func (l *meshListCmd) run() error {
 	}
 
 	w := newTabWriter(l.out)
-	fmt.Fprintln(w, "MESH NAME\tNAMESPACE\t")
+	nds, _ := l.clientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+
+	fmt.Fprintf(w, "OSM Deployments on cluster %s\n", nds.Items[0].GetName())
+	fmt.Fprintln(w, "\nMESH NAME\tNAMESPACE\tCONTROLLER PODS\tJOINED NAMESPACES\t")
 	for _, elem := range list.Items {
 		m := elem.ObjectMeta.Labels["meshName"]
 		ns := elem.ObjectMeta.Namespace
-		fmt.Fprintf(w, "%s\t%s\t\n", m, ns)
+		x := getNamespacePods(l.clientSet, m, ns)
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\n", m, ns, strings.Join(x["Pods"], ","), strings.Join(x["Joined Namespaces"], ","))
```

Comment:
How do we handle the case when there are 20-40 joined namespaces? This might not display well on the UI. I think we should add a `osm mesh describe` command that allows for more detailed info of a mesh like joined namespaces and keep the `osm mesh list` command less verbose and as a quick way to simply view all meshes in the cluster. What do you think?

Reply:
Hi @michelleN, I totally agree. Could we add a new column named `Status` and show the mesh-controller health status as, an alternative to adding the cluster name and joined namespace in this PR?

This can be useful when we check the mesh-controller status, some like:

```
MESH NAME          NAMESPACE                CONTROLLER PODS                   STATUS   
helm-install-osm   helm-install-namespace   osm-controller-8bc7fcc47-c5qrc    Pending
osm                osm-system               osm-controller-767c7995f8-skmp5   Running

```

Reply:
Hi @michelleN and @Aisuko thanks for the feedback, less verbose is very good I will make the required changes.
@Aisuko for the STATUS column if there are more than 1 controller pods then it will be confusing.

Reply:
+1 to the Status column. Once we have a describe command, we may want to talk about moving the Status column there but for now, this would be great.

Reply:
@michelleN @Aisuko so for now we should drop the idea for status column?

Reply:
yea that sounds good to me.